Package.swift

```diff
@@ -174,7 +176,10 @@ let package = Package(
         .target(
             /** High-level commands */
             name: "Commands",
-            dependencies: ["SwiftToolsSupport-auto", "Basics", "Build", "PackageGraph", "SourceControl", "Xcodeproj", "Workspace", "XCBuildSupport", "ArgumentParser"]),
+            dependencies: ["SwiftToolsSupport-auto", "Basics", "Build", "PackageGraph", "SourceControl", "Xcodeproj", "Workspace", "XCBuildSupport", "ArgumentParser"] + (buildPackageSyntax ? ["PackageSyntax"] : []),
+            swiftSettings: buildPackageSyntax ?
+                [.define("BUILD_PACKAGE_SYNTAX")] :
+                [.define("DONT_BUILD_PACKAGE_SYNTAX")]), // Unnecessary, but SPM errors if swiftSettings is empty.
```

Comment:
That seems like a bug that’s worth fixing.

Reply:
I think https://github.com/apple/swift-package-manager/pull/2880 will address this, it just hasn't made it into a release yet. In the meantime, it looks like `nil` is allowed, so I switched to that instead of the unused define

Reply:
Thanks.  Looks like this will be in 5.4, which SwiftPM doesn't require yet, but perhaps could require once 5.4 is released.